src/components/NcActionInput/NcActionInput.vue

```diff
@@ -385,6 +386,13 @@ export default {
 			type: Boolean,
 			default: true,
 		},
+		/**
+		 * Whether to append the `NcDateTimePicker` and `NcSelect` to body.
```

Comment:
```suggestion
		 * Whether to append the `NcSelect` to body.
```

Reply:
Now that you write that, I realized that this PR does change the behavior of the component, as it is now not possible anymore to append the datepicker to body (since I defined a prop now, it is not in `attrs` anymore which we bind).

However, appending the datepicker to body is broken anyway:
![image](https://github.com/nextcloud-libraries/nextcloud-vue/assets/2496460/c96a2f5a-e593-4091-bc41-e301b6a953df)

Should I anyway keep the possibility to append it to body and forward `appendToBody` to the datepicker as well?

Reply:
🤔 I would only forward it to NcSelect for now and then to datepicker after appendToBody is fixed for it

Reply:
With the solution from @ShGKme it will keep (not-)working as is for the datepicker.